packages/eslint-plugin-vtex/docs/rules/enforce-explicit-enum-values.md

```diff
@@ -0,0 +1,31 @@
+# Enforce explicit `enum` values (enforce-explicit-enum-values)
+
+`enum`s are a practical way to organize semantically related constant values. However, by implicitly defining values, `enum`s can lead to unexpected bugs if it's modified without paying attention to the order of its items.
+
+## Rule Details
+
+The following patterns are considered errors:
+
+```js
+const enum Status {
```

Comment:
`const enum` is a typescript feature (js doesn't have it)
```suggestion
```ts
const enum Status {
```